src/System.Private.CoreLib/src/System/Diagnostics/Eventing/DotNETRuntimeEventSource.cs

```diff
@@ -6,19 +6,19 @@ namespace System.Diagnostics.Tracing
 {
 #if FEATURE_PERFTRACING
```

Comment:
Could you please add `Condition` to the .csproj file, and remove this end-to-end ifdef.